doc/EFFECTS_JSON.md

```diff
@@ -230,11 +230,12 @@ future applications decreasing the overall time left.
 
 ### Intensities
 Intensities are used to control effect effects, names, and descriptions. They are defined with:
-```C++
-    "int_add_val": 2        - Defaults to 0! This means future applications will not increase intensity unless changed!
-    and/or
+```JSON
+    "int_add_val": 2         - Defaults to 0! This means future applications will not increase intensity unless changed!
+
     "int_decay_step": -2,    - Defaults to -1
-    "int_decay_tick": 10
+    "int_decay_tick": 10     - Defaults to 0, preventing intensity decay
```

Comment:
is this in seconds or turns or the amount decayed by each unit of time?

Reply:
Seconds, there's a bit-too-longwinded explanation below that section that I didn't try to integrate into the table. I'll add some notes to make it more obvious at a skim.